src/style.css

```
@@ -1 +1,69 @@
-/* Styles go here */
+* {
+  margin: 0;
+  padding: 0;
+}
+
+.header {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  flex-wrap: wrap;
+  height: 60px;
+  font-family: "Roboto", sans-serif;
+  padding: 0 50px;
+}
+
+.logo {
+  display: flex;
+  height: 40px;
+  width: 40px;
+}
+
+.nav {
+  display: flex;
+  align-items: center;
+}
+
+.nav-list {
+  display: flex;
+  list-style: none;
+}
+
+li {
+  margin-right: 20px;
+}
+
+li:last-child {
```

```suggestion
nav-item {
  margin-right: 20px;
}

nav-item:last-child {
```
Не потрібно використовувати селектори по тегам. В тебе ж є клас для li  (nav-item)